js_pt1/dest/desktop/js/kintoneRESTAPI.js

```diff
@@ -0,0 +1,47 @@
+(function() {
+	'use strict';
+	kintone.events.on('app.record.create.show', function(event) {
+		const params = {
+			'app': event.appId
+		};
+		kintone.api(kintone.api.url('/k/v1/app/form/fields'), 'GET', params).then(function(resp) {
+			const setRecords = kintone.app.record.get();
+			const actions = [];
+			const options = resp.properties.table.fields.action_5.options;
+			for (let i = 0; i < 6; i++) {
+				for (let key in options) {
+					if (options[key].index == i) {
```

Comment:
You should avoid using `==` as double equals in JavaScript behave in a very tricky way.
[JavaScript — Double Equals vs. Triple Equals - codeburst](https://codeburst.io/javascript-double-equals-vs-triple-equals-61d4ce5a121a)

You can write as follows instead.
```js
if (Number(options[key].index) === i) {
```